tests/acceptance/test_organization_events_v2.py

```diff
@@ -84,5 +90,75 @@ def test_errors(self, mock_now):
 
         with self.feature(FEATURE_NAME):
             self.browser.get(self.path + '?view=errors')
-            self.browser.wait_until_not('.loading-indicator')
-            self.browser.snapshot('events-v2 - errors')
+            self.wait_until_loaded()
+            self.browser.snapshot('events-v2 - error list')
```

Comment:
Do we need this snapshot? It's pretty similar to `events-v2 - all events`

Reply:
I could add more events so that there is more than one group.

Reply:
Ah right, could we add extra events to to the `events-v2 - errors` snapshot instead? That snapshot is supposed to capture this tab

Reply:
Sure.